ucsb-cs156-s22/s22-6pm-happycows
# Dj add enddate to commons

# Overview

Adds an endDate field to Commons backend
Note: I'm not sure why this branch includes laszlo's commits from her CRUD branch; I'll see if I can remove them.

# Issues Addressed

Addresses Issue #57 

# Details

The Commons backend api now returns an endDate for each commons, which could be used in the future to implement a countdown for the number of days left in a commons.

# Test Plan (for interactive testing)

* Load the page onto localhost:8080
* Go to the Swagger UI page
* Use sample PUT and GET requests to confirm that you get an endDate

Coverage:
![image](https://user-images.githubusercontent.com/72847482/171572922-e5dbfaa7-4713-4449-a3cc-b911076bf80d.png)

Mutation testing:
![image](https://user-images.githubusercontent.com/72847482/171572639-d92846ce-6425-45ea-987c-1ad92c227dd8.png)
